library/analytics/src/main/java/org/cru/godtools/analytics/AnalyticsService.java

```diff
@@ -118,6 +136,18 @@ private void trackScreenViewInAdobe(@NonNull final String screen) {
         }
     }
 
+    @AnyThread
+    private void trackScreenViewInSnowPlow(@NonNull final String screen)
+    {
+        final Activity activity = mActiveActivity != null ? mActiveActivity.get() : null;
+        if (activity != null) {
```

Comment:
it looks like you don't need to check for the presence of activity at all.

Reply:
Not sure why I did this, at some point I was thinking that the activity was required somewhere, maybe I was thinking of [here](https://github.com/snowplow/snowplow/wiki/Android-Tracker#2315-sets-up-an-activity-lifecycle-handler)

Reply:
k, let's drop the activity conditional around the task :)